ocs_ci/ocs/resources/pvc.py

```diff
@@ -140,6 +141,21 @@ def resize_pvc(self, new_size, verify=False):
             return self.size == new_size
         return True
 
+    def get_attached_pods(self):
+        """
+        Get the pods attached to this PVC
```

Comment:
if you are raising a new PR please add also:
"Get the pods attached to the PVC represented by this object instance

Reply:
yes, this PR became to loaded with different things, will raise a small pr

Reply:
yes, this PR became to much complecated, will raise a smaller pr with hellping funcs